src/peergos/server/corenode/NonWriteThroughCoreNode.java

```diff
@@ -13,23 +13,25 @@
 public class NonWriteThroughCoreNode implements CoreNode {
```

Comment:
should this class be threadsafe?

Reply:
Not really, it's mainly for local testing trying out migration code on real data.